### ISSUE TITLE
Cannot cancel the job

### ISSUE THREAD

**Issue by retifrav** [retifrav/rclone-rc-web-gui#4]
List of current transfers:

```
http://IP-ADDRESS:PORT/core/stats

{
    ...
    "transferring": [
        {
            "bytes": 212856832,
            "eta": 93,
            "group": "job/95180",
            "name": "2019-09-26-raspbian-buster-lite.zip",
            "percentage": 46,
            "size": 454279954,
            "speed": 2750878.5798009364,
            "speedAvg": 2595620.440967448
        },
        {
            "bytes": 191885312,
            "eta": 97,
            "group": "job/95191",
            "name": "some-file.mkv",
            "percentage": 42,
            "size": 446388267,
            "speed": 2537641.0826345356,
            "speedAvg": 2609023.91652754
        }
    ],
    ...
}
```

Want to cancel `some-file.mkv`, so the `jobid` is `95191`:

```
http://IP-ADDRESS:PORT/job/stop
{"jobid":"95191"}
```

Request is executed successfully, but the job is not cancelled.

I can also check its status:

```
http://IP-ADDRESS:PORT/job/status jobid="95191"

{
    "duration": 0,
    "endTime": "0001-01-01T00:00:00Z",
    "error": "",
    "finished": false,
    "group": "job/95191",
    "id": 95191,
    "output": null,
    "startTime": "2020-03-25T21:09:49.825654778Z",
    "success": false
}
```

What's weird is that on localhost (Windows 10 x64) it works fine, so the job gets cancelled. But when I do that on a remote host (Linux ARM), it does not.

**Comment by retifrav** [comment]
Asked the question at the [rclone forum](https://forum.rclone.org/t/rc-job-stop-doesnt-stop-cancel-the-transfer/15317).

**Comment by retifrav** [comment]
Some backends do not support "context propagation": https://github.com/rclone/rclone/issues/3257
So we can only wait for this to be implemented for all the backends.

**Comment by retifrav** [comment]
Looks like it has been fixed! I could successfully cancel an SFTP transfer with rclone v1.58, and most likely the fix was already available even in earlier versions (*since v1.55, as it seems*), as the upstream issue has been [closed on 03.02.2021](https://github.com/rclone/rclone/issues/3257#issuecomment-772633521).
